### PR TITLE
Fix language tag.

### PR DESCRIPTION
**Pull request by zaki50**
I removed some language label string in PR #154 but they are used in tags attached to the sessions.
I restored the language label strings for each languages.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -47,9 +47,21 @@
...
+    <string name="lang_en_in_en" translatable="false">English</string>
+    <string name="lang_ja_in_en" translatable="false">Japanese</string>
+
+    <string name="lang_ar_in_ja" translatable="false">アラビア語</string>
+    <string name="lang_en_in_ja" translatable="false">英語</string>
+    <string name="lang_ja_in_ja" translatable="false">日本語</string>
+
+    <string name="lang_ar">@string/lang_ar_in_en</string>
+    <string name="lang_en">@string/lang_en_in_en</string>
+    <string name="lang_ja">@string/lang_ja_in_en</string>
```
Comment (konifar) [inline app/src/main/res/values/strings.xml]: Just one question :pray: 
I guess it is enough. Why did you put all language to strings.xml?

values-ar/strings.xml
```xml
<string name="lang_ar_in_ar" translatable="false">العربية</string>
<string name="lang_en_in_ar" translatable="false">الإنجليزية</string>
<string name="lang_ja_in_ar" translatable="false">اليابانية</string>
```

values/strings.xml
```xml
<string name="lang_ar_in_en" translatable="false">Arabic</string>
<string name="lang_en_in_en" translatable="false">English</string>
<string name="lang_ja_in_en" translatable="false">Japanese</string>
```

values-ja/strings.xml
```xml
<string name="lang_ar_in_ja" translatable="false">アラビア語</string>
<string name="lang_en_in_ja" translatable="false">英語</string>
<string name="lang_ja_in_ja" translatable="false">日本語</string>
```

Comment (zaki50) [inline app/src/main/res/values/strings.xml]: I think there is no way to access the string value defined in other language than current locale.
If there is a way, you are right.

Comment (zaki50) [inline app/src/main/res/values/strings.xml]: And we need the way to access the values of different locate in SettingsFragment to fix #124

Comment (konifar) [inline app/src/main/res/values/strings.xml]: I see. Thanks! I have no other idea...

**Comment by konifar** [comment]
I guess your original master branch may be not up to date.
So could you try to merge master branch to your branch?

**Comment by zaki50** [comment]
@konifar I rebased the commits

**Comment by konifar** [comment]
Thanks! Hope CI success :pray:

**Comment by zaki50** [comment]
no success... :cry:

**Comment by konifar** [comment]
Maybe this is the reason.

```
> Building 36% > :app:lintVitalProductionRelease/home/ubuntu/droidkaigi2016/app/src/main/res/values/strings_no_trans.xml:2: Error: "app_name" is not translated in "ar" (Arabic), "ja" (Japanese) [MissingTranslation]
> Building 36% > :app:lintVitalProductionRelease    <string name="app_name">DroidKaigi 2016</string>
> Building 36% > :app:lintVitalProductionRelease            ~~~~~~~~~~~~~~~
```

It is my mistake. Could you change it to below? :bow: 

```xml
 <string name="app_name" translatable="false">DroidKaigi 2016</string>
```

**Comment by konifar** [comment]
I fixed CircleCI setting. So could you merge or rebase latest master branch to pass CI?

**Comment by konifar** [comment]
Yeah!! :clap: Thanks for your fix!

**Pull request merged**